commands/create_user

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+help() {
+  cat <<EOF
+Usage: $(basename "$0") [--username <username>] [--password <password>] [--email <email>]
+Ex: c8y oee \create_user --username abctest --password Password.123 --email dummy-oeeadmin@softwareag.com
+Create a user with global roles "OEE Admin" and "OEE Reader".
+EOF
+}
+
+echo "URL       :  $C8Y_HOST"
+echo "Tenant ID :  $C8Y_TENANT"
+echo "User      :  $C8Y_USER"
+
+# Set default values
+username="oeeadmin"
+password="Password.123"
+email="dummy-oeeadmin@softwareag.com"
```

Comment:
I'm not sure if we should use default values b/c the user will e.g. not know the default password. Instead we should throw an error if a value is missing.